pydicom/dataset.py

```diff
@@ -1656,10 +1657,10 @@ def compress(
             <pydicom.pixel_data_handlers>` to use to decompress it. If not
             specified then all available handlers will be tried (default).
         encapsulate_ext : bool, optional
-            If ``True`` then force the use extended encapsulation. If ``False``
-            (default) then extended encapsulation will be used
-            if needed for large amounts of compressed *Pixel Data*, but
-            will otherwise default to basic encapsulation.
+            If ``True`` then force the addition of an extended offset table.
+            If ``False`` (default) then an extended offset table
+            will be added if needed for large amounts of compressed *Pixel
+            Data*, but will otherwise default to just the basic offset table.
         **kwargs
```

Comment:
"but will otherwise" is unclear, maybe use something like
"otherwise just a basic offset tabele will be used."

Reply:
Done